Apps/Examples/Examples/All Examples/Annotations/AnimatedMarkerExample.swift

```diff
@@ -0,0 +1,117 @@
+import Foundation
+import MapboxMaps
+import UIKit
+
+@objc(AnimatedMarkerExample)
+final class AnimatedMarkerExample: UIViewController, ExampleProtocol {
+    enum Constants {
+        static let markerIconId = "marker_icon"
+        static let sourceId = "source-id"
+        static let animationDuration: CFTimeInterval = 2
+    }
+    private var mapView: MapView!
+
+    private var currentPosition = CLLocationCoordinate2D(latitude: 64.900932, longitude: -18.167040)
+
+    private var animationStartTimestamp: CFTimeInterval = 0
+    private var origin: CLLocationCoordinate2D!
+    private var destination: CLLocationCoordinate2D!
+    private var displayLink: CADisplayLink?
+
+    deinit {
+        displayLink?.invalidate()
+    }
```

Comment:
See other note about strong reference cycle. Maybe move this to viewDidDisappear or something like that?